popmon/visualization/utils.py

```diff
@@ -46,6 +47,48 @@ def xtick(lab, top):
     return lab
 
 
+def get_summary_table(
+    num_features, time_bins, time_axis, reference_type, online_report
+):
+
+    data = {
+        "date generated": datetime.datetime.now().strftime("%Y-%m-%d %H:%M:%S"),
+        "time axis": time_axis,
+        "reference type": reference_type,
+        "number of features": num_features,
+        "Time bins": time_bins,
+        "online report": online_report,
```

Comment:
Any reason why most keys start with a lowercase letter and one with an uppercase?